source/rofi.c

```diff
@@ -951,78 +951,157 @@ static void calculate_window_position ( MenuState *state, const workarea *mon )
     state->y += config.y_offset;
 }
 
+/**
+ * @param line_height The uniform height of a line of text.
+ *
+ * @returns Number of lines contained in the menu.
+ */
+static int menu_num_lines ( int line_height ) {
+    workarea mon;
+    monitor_active ( &mon );
```

Comment:
960 and 961 should be moved within the if statement (964) as they are only required in that scope.